src/stores/user.js

```diff
@@ -16,8 +17,10 @@ export const useUserStore = defineStore({
       const userRef = collection(firebaseStore, collections.user);
       const q = query(userRef,where('uid','==',uid));
       const querySnapshot = await getDocs(q);
+      const docId = querySnapshot.docs[0].id; // gets the document id
       if (querySnapshot.size === 1) {
         this.user.info = querySnapshot.docs[0].data();
+        this.user.docs = docId;
```

Comment:
Can store the query snap shot as this.user.doc = querySnapshot.docs[0]. Now if you need the doc id, it is just this.user.doc.id

Reply:
So like this?
![image](https://user-images.githubusercontent.com/46050460/233812065-5dbb27e4-76c8-4ade-b69e-dca4f1402c12.png)

Reply:
yes